src/ast.cpp

```diff
@@ -2226,4 +2226,20 @@ namespace Sass {
     }
   }
 
+  //////////////////////////////////////////////////////////////////////////////////////////
+  // Convert map to (key, value) list.
+  //////////////////////////////////////////////////////////////////////////////////////////
+  List* Map::to_list(Context& ctx, ParserState& pstate) {
```

Comment:
As an aside it bugs that the AST doesn't have access to the context. Not being about to create new nodes from AST methods gets rather painful. In this case it might be nicer to add a `List` constructor that takes a `Map` argument.